server/pom.xml

```diff
@@ -215,6 +276,31 @@
 
   <build>
     <plugins>
+      <!-- Creates (or re-creates) a keywhizdb_test database. -->
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>sql-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>create-keywhiz-database</id>
+            <phase>process-test-classes</phase>
+            <goals>
+              <goal>execute</goal>
+            </goals>
+          </execution>
```

Comment:
The ID is an addition beyond the original configuration in model/pom.xml. It's necessary so that we can match the profile configuration with this specific execution.

Also, unlike Ant, Maven doesn't have an explicit notion of before/after hooks for each lifecycle phase, so if we need to do some pre-test setup we just need to arbitrarily pick one of the preceding phases and slap this thing in there.